Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/BreadCrumbs.cpp

```diff
@@ -291,31 +349,56 @@ namespace AzQtComponents
             htmlString.prepend(QStringLiteral("%1%1%2%1%1").arg(nonBreakingSpace, arrowCharacter));
         };
 
-        // last section is not clickable
-        if (!m_truncatedPaths.isEmpty())
+        if (m_truncatedPaths.isEmpty())
         {
+            m_label->clear();
+            return;
+        }
+        else
+        {
+            // last section is not clickable
             plainTextPath = m_truncatedPaths.takeLast();
         }
```

Comment:
Nit: Don't use `else` after `return`
```suggestion
        if (m_truncatedPaths.isEmpty())
        {
            m_label->clear();
            return;
        }

        // last section is not clickable
        plainTextPath = m_truncatedPaths.takeLast();
```

Reply:
done